src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -743,21 +744,23 @@ temperature_hist_data <- map(temperature_hist_plots, ~ ggplot_build(.x)$data[[1]
 
 generate_seasonal_histogram <- function(df) {
   
-  winter <- c(12,1,2)
-  spring <- c(3,4,5)
-  summer <- c(6,7,8)
-  fall <- c(9,10,11)
-  seasons <- data.frame(winter, spring, summer, fall)
+  winter_baseflow <- c(12,1,2,3)
+  snowmelt <- c(4,5,6,NA)
```

Comment:
What are these NAs for?

Reply:
They make each vector the same length so that they play nice when they are combined into a df. There shouldn't be any NA's in the month column since its derived from the DT_round column so I don't think there should be any issues with this...

Reply:
gotcha, makes sense